packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -80,7 +78,7 @@ export class HttpPlugin extends classes.BasePlugin {
       shimmer.wrap(
           moduleExports && moduleExports.Server &&
               moduleExports.Server.prototype,
-          'emit', this.patchIncomingRequest());
+          'emit' as never, this.patchIncomingRequest());
```

Comment:
Why this change?

Reply:
This change is a workaround to solve a strange type cast error for this.patchIncomingRequest when we set the type for moduleExports as HttpModule.

Reply:
Regarding method visibility, we can not set patch* as private because subclass https needs access to those methods - they were set to protected instead.

Reply:
Would this work?

```js
shimmer.wrap(moduleExports.Server.prototype, 'emit', /*...*/);
```

Reply:
Could not make it work.

Reply:
Ok. Might be a problem with the shimmer type definitions.